### DISCUSSION TITLE
Panel 1 (Big picture) theme

### DISCUSSION THREAD

**Discussion by kjappelbaum** [MADICES/MADICES-2022#18] [Ideas]
We need to come up with a/some general theme(s) for the panel. 

Some ideas: 

- What exactly is the problem (lack of tooling, lack of awareness, lack of standards, lack of incentives) 
  - What did other communities do (e.g., bioinformatics) to address these problems? 
- To what extent is interoperability feasible? 
 - There is always a need for agility
 -  Experimental systems are rarely fully specified. Often, things are uncertain

**Comment by ml-evs** [comment]
Potential final "big picture" questions for the panel:

- What new research would you do if everyone was able to publish/ingest semantic data in our field?
- Efforts around COVID data/workflow streamlining between groups, countries etc. showed both the effort required and value obtained from doing this work on a "war footing". Do we need to take a similar approach in materials/chemistry to accelerate fundamental science for global challenges?

**Reply by kjappelbaum** [reply]
- How can we make sure that this will be still alive (in some form) tens of years from now. If the NFDIs reach end of funding probably the software will no longer be maintained. If we start something and find a real job, we won't have time to actively maintain it anymore.

**Reply by kjappelbaum** [reply]
- new research: https://twitter.com/KaiyaYuji/status/1482680466130235394 I think one can also think of mapping ELN directly to robot, then directly to process. That is, reproduce/test one's manual synthesis recorded in some lab notebook using lab automation.

**Comment by ml-evs** [comment]
Trying to hash this out as a few concrete questions:

1. What is your "pitch" to researchers to encourage them to engage with and improve data management practices? What are the biggest tangible changes (in funding, tooling, training, research practices) that you think could accelerate adoption? What has changed in the last 5/10/20 years in this regard?
1. **Who** are **you** doing interoperability for, what motivates you? Future researchers reading your work, collaborators, your own institute?
1. What new research would be enabled if, say, linked open data, or well-described APIs for automated labs, became ubiquitous in chemistry & materials science? How would the research landscape fundamentally change?
1. How do you see the open data ecosystem interacting with legacy publishing going forward? Who should be curating SI/data archives? What role does text mining/NLP have to play?
1. Efforts around COVID data/workflow streamlining between groups, countries etc. showed both the effort required and value obtained from doing this interoperability work on a "war footing". Do we need to take a similar approach in materials/chemistry to accelerate fundamental science for global challenges? Can we be sure that this does not incur too large an opportunity cost?
1. Are there any particular exemplary tools/initiatives (in any field) you wish to highlight?

**Comment by kroenlein** [comment]
(Copied from Zoom chat):
With my work with industry, while we always have some enthusiastic early adopters, the majority of material scientists in a given organization are very allergic to code.  Those staff are also very important people to extract and structure knowledge from, because those are the folks that have developed sophisticated intuition for how to build systems, even if they cannot articulate it.  Can you propose strategies for engaging with those colleagues?
